v2/emailpassword/serverless/with-aws-lambda/backend-config.mdx

```diff
@@ -9,25 +9,35 @@ import CoreInjector from "/src/components/coreInjector"
 
 # 2. Backend config
 
-## 1) Install supertokens package
-```bash
-npm i supertokens-node
-```
+## 1) Add layers for supertokens and middy
 
-## 2) Create a configuration file (`config.js`)
-- Create a `config.js` file in the root directory of your project.
-- An example of this file can be found [here](https://github.com/supertokens/supertokens-auth-react/blob/master/examples/with-aws-lambda/backend/config.js).
+- Download the zip files here
+1. supertokens -> https://github.com/saurabhghatnekar/supertokens-aws-lambda/raw/main/supertokens-node.zip
+2. @middy/core -> https://github.com/saurabhghatnekar/supertokens-aws-lambda/raw/main/middy-core-node.zip
+3. @middy/http-cors -> https://github.com/saurabhghatnekar/supertokens-aws-lambda/raw/main/middy-http-node.zip
+
+- Now, head to AWS Lambda.
+As shown in the image below, click on layers
+[image](https://user-images.githubusercontent.com/20884782/190074706-3d1b6420-e812-48d9-aa04-8ea2d4a4c62d.png)
+
+- Then, click in "Create Layer" button
+![image](https://user-images.githubusercontent.com/20884782/190075143-57768a61-7efc-4b3d-a865-0d68e84f3005.png)
+
+- Give a name for your layer, upload the zip and select the nodejs 16 runtime
+![image](https://user-images.githubusercontent.com/20884782/190076214-47e1ba13-3f17-4dcb-afc9-90b3e8c0f53c.png)
 
+- Repeat this for all the 3 modules.
 
-## 3) Create a backend config function
+## 2) Create a new Lambda Function
+- Create a new lambda function with Node 16 runtime
 
-<AppInfoForm
-    askForAppName
-    askForAPIDomain
-    askForWebsiteDomain
->
 
-<CoreInjector>
+## 3) Create a configuration file (`config.js`)
+- Create a `config.js` file in the root directory of your project.
+- An example of this file can be found [here](https://github.com/supertokens/supertokens-auth-react/blob/master/examples/with-aws-lambda/backend/config.js).
+
+
+## 4) Create a backend config function
 
 ```tsx title="config.ts"
```

Comment:
This needs to be wrapped around `AppInfoForm` and `CoreInjector` as it previously was